web/package.json

```diff
@@ -83,10 +83,10 @@
     "typedoc": "^0.23.25",
     "typedoc-plugin-markdown": "^3.14.0",
     "typescript": "4.7.4",
-    "webpack": "^5.94.0",
-    "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.6.0",
-    "webpack-node-externals": "^3.0.0"
+    "webpack": "5.75.0",
```

Comment:
that's a downgrade, is it necessary?

Reply:
I can double check on that one but it aligns with OCP console version + sdk one